wurst/systems/spells/mage/elementalist/Brambles.wurst

```diff
@@ -198,7 +198,7 @@ class BramblesInstance
                             let dummy = new DummyCaster()
                                 ..owner(owner)
                                 ..origin(ownerUnit.getPos())
-                            if dummy.castTarget(DUMMY_ENTANGLE_SPELL_ID, 1, Orders.entanglingroots, u)
+                            if dummy.castTarget(DUMMY_ENTANGLE_SPELL_ID, 1, Orders.entanglingroots, u) != null
```

Comment:
Why is this in here? You may need to merge master into this branch.

Reply:
Well this was before bramble update was merged I guess, I already merged master by now, you me to make it so this file isn't included in this pr?

Reply:
Did you merge master or bramble branch? You don't have the typical default message for a merge from master, e.g.

```
Merge branch 'master' into bush-selection
```

Reply:
I think I did "git checkout master bramble.wurst", since I just needed 1 file

Reply:
Git views that as you changing the file, rather than just referencing someone else's file. I think the merge would be seamless for you, without any conflicts.